scripts/1.TCGA-MLexample.py

```diff
@@ -283,8 +300,13 @@ def get_threshold_metrics(y_true, y_pred):
 ax = sns.distplot(predict_df.query("status == 1").decision_function, hist=False, label='Positives')
 
 
-# In[30]:
+# In[32]:
 
 ax = sns.distplot(predict_df.query("status == 0").probability, hist=False, label='Negatives')
 ax = sns.distplot(predict_df.query("status == 1").probability, hist=False, label='Positives')
 
+
```

Comment:
When you think you're done with everything, clean up for final commit:

+ remove this empty cell
+ restart notebook and run all so all cells are in sequence from 1 to n.
+ rexport to script